tests/62attachments/01unicode.pl

```diff
@@ -0,0 +1,70 @@
+test "Can upload with unicode file name",
+    requires => [qw(first_v1_client user)],
+    provides => [qw(unicode_content_id)],
```

Comment:
"fixed" by extracting the content IDs to lexical variables instead of using test env.